lte/gateway/python/magma/pipelined/ng_manager/session_state_manager.py

```diff
@@ -0,0 +1,266 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import logging
+import copy
+from collections import (
```

Comment:
Here and in a few other places can you do newline args with just 1 tab, looks easier to read
```
from collections import (
    OrderedDict,
    namedtuple)

Reply:
Comments addressed.